auth-sample/build.gradle

```diff
@@ -25,8 +25,17 @@ android {
     compileSdkVersion 28
     buildToolsVersion "28.0.3"
 
+    signingConfigs {
+        debug {
+            storeFile file('keystore/example.keystore')
+            storePassword 'example'
+            keyAlias 'example_alias'
+            keyPassword 'example'
+        }
+    }
+
     defaultConfig {
-        applicationId "com.spotify.sdk.android.authentication.sample"
+        applicationId "com.spotify.android.appremote.demo"
```

Comment:
should this contain `appremote.demo`?

Reply:
Unfortunately yes, for now... I need to change the app entries in the developer console but haven't been able to do it due to permissions.
Essentially, we need to ensure that the app-id/key-ginfeprint pair is approved for the given client-id.
Alternatively I could create a new client-id for the old package-id.
What do you think it's better?

Reply:
Actually I was able to change the configuration. It's back to `com.spotify.sdk.android.authentication.sample` with our correct key.